Makefile

```diff
@@ -43,15 +43,23 @@ bin/version.h: VERSION.full
 bin/version.sh: VERSION.full
 	echo "version () { echo 1>&2 '$$(cat $<)'; }" > $@
 
+# Clone BATS if it's not already there
```

Comment:
Bats is a Git submodule, so these should be `git submodule` commands rather than `clone`.

I am pleased to have it encapsulated somehow though, since I never remember the proper syntax.